tutorials/algorithms/11_gradients_framework.ipynb

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
<p>On a second look, I think that the QFI should have its own blue box, at the same level as Base QGT and Circuit QGTs. Apart from that, this figure is great!!!</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1463/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
As both QFI and QGT extend from the BaseQGT class, I wanted to keep the ideas connected.

Reply:
I see what you mean, but the QFI class is not a subclass of BaseQGT, it uses an instance of it, which is different. If you still want to keep the ideas connected, you could maybe change the title to "Circuit QGTs and QFI"